tests/messages.test.ts

```diff
@@ -64,6 +66,42 @@ describe('Messages', () => {
 		});
 	});
 
+	describe('maskedLink', () => {
+		test('GIVEN "https://discord.js.org" THEN returns "<https://discord.js.org>"', () => {
+			expect<'<https://discord.js.org>'>(maskedLink('https://discord.js.org')).toBe('<https://discord.js.org>');
+		});
+
+		test('GIVEN new URL("https://discord.js.org") THEN returns "<https://discord.js.org>"', () => {
+			expect<`<${string}>`>(maskedLink(new URL('https://discord.js.org/'))).toBe('<https://discord.js.org/>');
+		});
+	});
+
+	describe('hyperlink', () => {
+		test('GIVEN content and string URL THEN returns "[content](url)"', () => {
+			expect<'[discord.js](https://discord.js.org)'>(hyperlink('discord.js', 'https://discord.js.org')).toBe(
+				'[discord.js](https://discord.js.org)',
+			);
+		});
+
+		test('GIVEN content and URL THEN returns "[content](url)"', () => {
+			expect<`[discord.js](${string})`>(hyperlink('discord.js', new URL('https://discord.js.org'))).toBe(
+				'[discord.js](https://discord.js.org/)',
+			);
+		});
+
+		test('GIVEN content, string URL, and title THEN returns "[content](url title)"', () => {
```

Comment:
Same as above. Lines below should be changed as well.
```suggestion
		test('GIVEN content, string URL, and title THEN returns "[content](url \"title\")"', () => {
```